mvc/models/removeComment.js

```diff
@@ -0,0 +1,27 @@
+const db = require("../../db/connection.js");
+
+const removeComment = async (id) => {
+  console.log("triggered");
+  try {
+    const checkCommentExists = await db.query(
+      "SELECT 1 FROM comments WHERE comment_id=$1",
+      [id]
+    );
```

Comment:
we could do this in one query rather than two to make the code more 'dry' - definitely worth a refactor in your own time but nice work